x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -834,6 +847,30 @@ describe('AllCasesListGeneric', () => {
       expect(checkbox).not.toBeChecked();
     }
 
-    waitForComponentToUpdate();
+    await waitForComponentToUpdate();
+  });
+
+  it('should hide the alerts column if the alert feature is disabled', async () => {
```

Comment:
Can you please add a case for rendering the alerts column if you pass an empty alerts object and no alerts object?